webscalesqlclient/CMakeLists.txt

```diff
@@ -2,6 +2,7 @@ cmake_minimum_required(VERSION 2.8.0)
 include(ExternalProject)
 ExternalProject_Add(
   webscalesqlclient
+  URL file://${CMAKE_CURRENT_SOURCE_DIR}/webscalesql-5.6
```

Comment:
In particular, since this is a submodule, we already have implicit information in how to download and get the exact version you need. Telling CMake "you can download this from the location it's already downloaded" seems redundant and superfluous -- and has the potential to lead to trouble? What is the URL used for anyways?